src-tauri/src/binaries/adapter_tor.rs

```diff
@@ -71,7 +71,14 @@ impl LatestVersionApiAdapter for TorReleaseAdapter {
             .join(format!("{}.asc", asset.name));
         let checksum_url = format!("{}.asc", asset.url);
 
-        match download_file_with_retries(&checksum_url, &checksum_path, progress_tracker).await {
+        let user_agent = format!(
+            "universe {}({}) | {}",
+            env!("CARGO_PKG_VERSION"),
+            std::env::consts::OS,
+            "github adapter"
```

Comment:
Should this say "tor adapter" ?

```suggestion
            "tor adapter"
```

Reply:
Agree, only TU version and platform info I think is useful. Getting reports on TU version or platform usage from CDN provider is possible. Different ```adapter``` would not request resources from different services.